### PR TITLE
Add Sepa Conformity

### PR DESCRIPTION
**Pull request by gigo1980**

### CODE REVIEW

**Comment by stoilkov** [comment]
@gigo1980 is mandate_reference not also part of the transaction response ( lib/Paymill/Models/Response/Transaction.php ) ?

**Comment by gigo1980** [comment]
Bin in 15 min im büro
Am 16.03.2015 09:05 schrieb "Emilian Stoilkov" <notifications@github.com>:

**Comment by gigo1980** [comment]
And. Yes it should be
Am 16.03.2015 09:05 schrieb "Emilian Stoilkov" <notifications@github.com>:

**Comment by stoilkov** [comment]
Ok, we have to add the mandate_reference to the Transaction Response before merging.

File: lib/Paymill/Models/Request/Subscription.php [hunk 1]
```diff
@@ -475,6 +504,9 @@ public function parameterize($method)
                 if (!is_null($this->getOfferChangeType())) {
                     $parameterArray['offer_change_type'] = $this->getOfferChangeType();
                 }
+                if (!is_null($this->getMandateReference())) {
```
Comment (stoilkov) [inline lib/Paymill/Models/Request/Subscription.php]: I think that's the same lines twice

**Pull request merged**